tools/c7n_azure/c7n_azure/provisioning/function_app.py

```diff
@@ -23,26 +21,32 @@ def _provision(self, params):
         site_config = SiteConfig(app_settings=[])
         functionapp_def = Site(location=params['location'], site_config=site_config)
 
+        # common function app settings
         functionapp_def.kind = 'functionapp,linux'
+        functionapp_def.reserved = True

```

Comment:
some comment to explain why it is required?

Reply:
It means Linux

Reply:
Looks like it works without explicitly setting it now.